WalletWasabi/Stores/IndexStore.cs

```diff
@@ -251,7 +266,7 @@ public async Task AddNewFiltersAsync(IEnumerable<FilterModel> filters)
 	{
 		using (await IndexLock.LockAsync(CancellationToken.None).ConfigureAwait(false))
 		{
-			await using SqliteTransaction sqliteTransaction = IndexStorage.BeginTransaction();
+			using SqliteTransaction sqliteTransaction = IndexStorage.BeginTransaction();
```

Comment:
`SqlLiteTransaction` is `IAsyncDisposable`. Is there a reason to remove the `await` here? This way we're using the sync dispose API.